### DISCUSSION TITLE
Ark UI Plus licence

### DISCUSSION THREAD

**Discussion by nikme** [chakra-ui/ark#2524] [General]
https://ark-ui.com/react/plus

the page https://ark-ui.com/license currently throws 404,
But I'm interested in the difference between licenses. Like, if I have a small company and work alone, do I need to get a personal or a team license? 
What components do we get with the plus?
br

**Comment by segunadebayo** [comment]
Hi @nikme, thanks for pointing this out. We'll get it resolved ASAP

You'll need to get a personal license since you work alone.

Ark Plus will include complex component compositions that might otherwise take a while to build on your own, you also get to tell us what compositions you want to see.

**Reply by nikme** [reply]
hi, 
So, 2 more issues - once licence bought redirect is to 404
If you try to sign in, URL to which you are redirected is http://n/. Token is not saved and there is no way to get to pro components. 
br

**Reply by cschroeter** [reply]
@nikme 

This appears to be an issue with Vercel at the moment. We are kinda unable right now. Please feel free to reach out to me on Twitter and I send you the snippets. Sorry for the inconvenience. Hope that this is sorted out shortly.

**Update:**

Should be fixed by now. Sorry for the inconvenience and thanks again for the support <3
Just dropped a `cmdk` component for you ;)
